contrib/universal-ctags/template.py

```diff
@@ -0,0 +1,37 @@
+pkgname = "universal-ctags"
+pkgver = "6.1.0"
+pkgrel = 0
+build_style = "gnu_configure"
+configure_args = [
+    "--enable-iconv",
+    "--enable-json",
+    "--enable-pcre2",
+    "--enable-xml",
+    "--enable-yaml"
+]
+make_cmd = "gmake"
+make_dir = "."
+hostmakedepends = [
+    "automake",
+    "gmake",
+    "jansson-devel",
+    "libtool",
+    "libxml2-devel",
+    "libyaml-devel",
+    "pcre2-devel",
+    "pkgconf",
+    "python-docutils"
+]
+depends = [
+    "jansson",
+    "libxml2",
+    "libpcre2",
+    "libyaml"
+]
+checkdepends = ["python"] + depends
```

Comment:
```suggestion
checkdepends = ["python"]
```

these are all traced by NEEDED from the elf file:  

```
=> universal-ctags-6.1.0-r0: running pre_pkg hook: 001_runtime_deps...
   SONAME: libxml2.so.2 <-> libxml2
   SONAME: libjansson.so.4 <-> jansson
   SONAME: libyaml-0.so.2 <-> libyaml
   SONAME: libpcre2-8.so.0 <-> libpcre2
```

Reply:
oh, very nice!